win32_event_log/datadog_checks/win32_event_log/constants.py

```diff
@@ -9,11 +9,15 @@
 #
 # However, event viewer & the C api do not show the constants from above, but rather return these:
 # https://docs.microsoft.com/en-us/windows/win32/wes/eventmanifestschema-leveltype-complextype#remarks
-EVENT_TYPES = {
-    'success': 4,
-    'error': 2,
-    'warning': 3,
-    'information': 4,
-    'success audit': 4,
-    'failure audit': 2,
+EVENT_TYPES_TO_LEVEL = {'success': 4, 'error': 2, 'warning': 3, 'information': 4}
+
+# Audit success and audit failures event do not correspond to a specific level but can instead be queries
+# by using specific keywords.
+# https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.eventing.reader.standardeventkeywords?view=netframework-4.8
+EVENT_TYPES_TO_KEYWORD = {
+    'failure audit': '0x8010000000000000',
+    'success audit': '0x8020000000000000',
+    # Aliases for legacy version
+    'audit failure': '0x8010000000000000',
+    'audit success': '0x8020000000000000',
```

Comment:
Is this just for user familiarity?

Reply:
Yes, as requested here: https://github.com/DataDog/integrations-core/pull/8596#issuecomment-778948031

Reply:
@ian28223 I would really prefer that we do not support legacy Windows terminology, they will see that right ones that we document:

> ##   - type > https://docs.microsoft.com/en-us/windows/win32/eventlog/event-types

wdyt?

Reply:
It would indeed be ideal if all users would open those links and choose from there but those who've been used to the legacy mode (have seen or are using the example from legacy) - or have a combination both legacy and non legacy style instances (I've seen a couple) - will have a tendency to copy from it.
- https://github.com/DataDog/integrations-core/blob/7.25.x/win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example#L268

We could add the expected ones as (examples here)[https://github.com/DataDog/integrations-core/blob/7.25.x/win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example#L111L113]  increasing the likelihood of users copying from it... and/or have the old ones emit some sort of deprecation notice if used.

Reply:
Ok following up with that discussion:
1. I updated the config to increase the likelihood of users using the correct name.
2. I removed the legacy naming.
3. If a user is using the legacy name, the check will fail explicitly with a ConfigurationError exception. This will make it extra clear

Reply:
LGTM! 👍